CONTRIBUTING.md

```diff
@@ -31,7 +31,14 @@ If installation fails due to a `node-gyp` issue you may need to perform some add
 
 ### on macOS
 
-- I have no idea.
+- Determine if you have Xcode command line tools installed
+  - example the console will tell you if they're already installed): `xcode-select --install`
```

Comment:
Can you make the steps more prescriptive? i.e., tell the user what to do directly.

Reply:
This style (minus the missing "(") matches the other instructions. Would you like me to modify those too? They're short so it wouldn't take long.